### ISSUE TITLE
For those who have tried, do you feel the 512 version is an improvement?

### ISSUE THREAD

**Issue by paulescobar** [neuralchen/SimSwap#171]
I am converting on mostly SD or sub-HD (720p) target content. I find the new "--crop_size 512" looks very strange. You can see borders on the edges of the new face & sometimes there are strange fluctuations (like wind is blowing beneath the face). The key facial features like eyes, nose, and mouth look a little more "darker" or "contrasty"...and I think the facial accuracy is lesser as a result. 

Could this be the result of increased sharpness in this version?

Honestly, I find setting it to "--crop_size 224" produces far better results. It blends properly & looks more accurate. I think this is because it has a "softer" look - which may help produce better accuracy.

**Comment by AaronPorts** [comment]
I think the problem is in the VGGFace2-HQ dataset, which is more contrasting and a little unnatural due to the resolution upscale.

**Comment by nonlin** [comment]
> 

Also partial? Didn't they say training on that is in beta or incomplete?

**Comment by lhucklen** [comment]
also curious as to if this will be improved upon.

**Comment by illtellyoulater** [comment]
I also agree that "--crop_size 224" produces better results. The new model seems to produce more detailed results, but at the same time it will look off and more unnatural.

**Comment by acSpock** [comment]
I'm pretty sure that one needs to train the 512 model themselves on top of the base 512. Don't believe there exists a complete model available, unless someone could correct me.
